modules/ROOT/pages/apikit-graphql-api-mapping.adoc

```diff
@@ -0,0 +1,59 @@
+= Mapping a GraphQL API to your Data Sources
+ifndef::env-site,env-github[]
+include::_attributes.adoc[]
+endif::[]
+
+== Select Sources
+
+The module sources vary depending on the implementation of your GraphQL API. APIkit for GraphQL provides two kind of sources:
+* Data Fetchers
+* Data Loaders
+Your selection depends on your implementation needs. 
+In addition, GraphQL is transport-layer agnostic so you choose the main connector listener depending on the protocol that you want to serve your data.
+
+=== Add a Listener
+
+The inbound endpoint that triggers the main application flow is the connector listener specific to the client-server protocol of your choice. It is common that GraphQL communicates over HTTP, so APIkit scaffolder generates an HTTP listener by default, but you can choose another source for serving your data over a different protocol.
+
+=== Add a Data Fetcher
+
+Data fetchers are sources of Mule flows. They fulfill the data of a single-object field in a query result. The combination of the following key parts identify them:
+
+* *Object type name*: The name of one of the object types in your schema.
+* * Field name*: The name of one of the fields that are part of the object type.
+
+You can add data fetchers for every object field defined on the schema, regardless of the field type. If the Mule application does not have configured a data fetcher for a field, the parent type populates the field value. Otherwise, the data the client requested is not fulfilled, so it results in *null*.
```

Comment:
```suggestion
You can add data fetchers for every object field defined on the schema, regardless of the field type. If the Mule application does not have a data fetcher configured for a field, the parent type populates the field value. Otherwise, the client-requested data is not fulfilled, resulting in *null*.
```

Reply:
Thanks!